tests/utils_tests/testing_tests/assertions_tests/test_assert_is_point.py

```diff
@@ -6,70 +6,88 @@
 from chainercv.utils import assert_is_point
 
 
-def _random_mask_including_true(n):
+def _random_visible_including_true(n):
     while True:
-        mask = np.random.randint(0, 2, size=n).astype(np.bool)
-        if mask.any():
-            return mask
+        visible = np.random.randint(0, 2, size=n).astype(np.bool)
+        if visible.any():
+            return visible
 
 
 @testing.parameterize(
-    # no mask and size
-    {'point': np.random.uniform(-1, 1, size=(10, 2)).astype(np.float32),
-     'valid': True},
-    {'point': ((1., 2.), (4., 8.)),
-     'valid': False},
-    {'point': np.random.uniform(-1, 1, size=(10, 2)).astype(np.int32),
-     'valid': False},
-    {'point': np.random.uniform(-1, 1, size=(10, 3)).astype(np.float32),
-     'valid': False},
-    # use mask, no size
-    {'point': np.random.uniform(-1, 1, size=(10, 2)).astype(np.float32),
-     'mask': np.random.randint(0, 2, size=(10,)).astype(np.bool),
-     'valid': True},
-    {'point': np.random.uniform(-1, 1, size=(4, 2)).astype(np.float32),
-     'mask': (True, True, True, True),
-     'valid': False},
-    {'point': np.random.uniform(-1, 1, size=(10, 2)).astype(np.float32),
-     'mask': np.random.randint(0, 2, size=(10,)).astype(np.int32),
-     'valid': False},
-    {'point': np.random.uniform(-1, 1, size=(10, 2)).astype(np.float32),
-     'mask': np.random.randint(0, 2, size=(10, 2)).astype(np.bool),
-     'valid': False},
-    {'point': np.random.uniform(-1, 1, size=(10, 2)).astype(np.float32),
-     'mask': np.random.randint(0, 2, size=(9,)).astype(np.bool),
-     'valid': False},
-    # no mask, use size
-    {'point': np.random.uniform(0, 32, size=(10, 2)).astype(np.float32),
+    # no visible and size
+    {'point': np.random.uniform(-1, 1, size=(1, 10, 2)).astype(np.float32),
+     'no_error': True},
```

Comment:
Why did you replace `valid` with `no_error`?

Reply:
Actually, I don't need to anymore.
I changed it initially because I was using `valid` as a variable name instead of `visible`.